### ISSUE TITLE
Update front-matter

### ISSUE THREAD

**Issue by oscarlevin** [OpenDiscreteMath/ibl-combinatorics#90]
We should probably have a new preface briefly describing the conversion we did.  Then retitle the current preface something like "Preface to original edition".

We will also need a cover, copyright page, etc.  I'm thinking of treating this as if we are a new publisher who has obtained the copyright to an old book, spruces up the layout and releases it.  I should look at some dover books to see what they do about copyright.  Perhaps, something like "original copyright 2004 Bogart, this edition, copyright 2017 Open Discrete Math."

**Comment by mitchkeller** [comment]
Definitely yes to the preface bits. We will have to decide what to do with the part of the original preface that refers to the problem decorations. I'm thinking just edit to make sense in light of how we're displaying that information and fess up to it in our preface, but I'm open to other things.

In terms of copyright, we will have to think a little bit about how to phrase that. The bulk of the copyright will remain with Bogart's heirs, but there certainly is an element of it that will need specification. Dover usually pulls their stuff off because authors get the copyright back from the publisher (or never relinquished it, and the publisher's rights expire or they surrender them). Grabbing a random Dover book off my shelf, I found a verso that says:

> Bibliographical Note
> This Dover edition, first published in 2002, is an unabridged republication of the work originally published in 1967 by John Wiley & Sons, Inc., New York.

We certainly can do something to that effect, and "unabridged" would apply here. I have taken the liberty in a few places of correcting errors that were noted in my hard copy of the book, and I think we can say that in more detail in the preface but perhaps also note something as part of this notice that we've made such updates (and welcome notification of others!).
